a10_neutron_lbaas/a10_config.py

```diff
@@ -41,29 +41,29 @@ def __init__(self, config_dir=None):
             d = '/etc/neutron/services/loadbalancer/a10networks'
         else:
             d = '/etc/a10'
-        self.config_dir = os.environ.get('A10_CONFIG_DIR', d)
-        self.config_path = os.path.join(self.config_dir, "config.py")
+        self._config_dir = os.environ.get('A10_CONFIG_DIR', d)
```

Comment:
This code chooses the first of:
A10_CONFIG_DIR
passed in
venv
crappy legacy path
/etc/a10